Snowflake.Data/Logger/SFLoggerFactory.cs

```diff
@@ -1,51 +1,81 @@
-﻿/*
+/*
  * Copyright (c) 2012-2019 Snowflake Computing Inc. All rights reserved.
  */
 
-using log4net;
+using Microsoft.Extensions.Logging;
+using System.IO;
 
 namespace Snowflake.Data.Log
 {
-    class SFLoggerFactory
+    public class SFLoggerFactory
     {
-        private static bool isLoggerEnabled = true;
+        private static bool s_isCustomLoggerEnabled = false;
 
-        private static SFLogger logger = null;
+        private static bool s_isSFLoggerEnabled = true;

```

Comment:
shouldn't it be disabled by default? I thought Easy Logging feature would enable it?